cpp/ps0001/p1Solution.cpp

```diff
@@ -1,19 +1,17 @@
-#include<iostream>
-using namespace std;
-double CalculateSum(double num1, double num2, double num3) {
-	return num1 + num2 + num3;
-}
-int main() {
+#include <iostream>
 
-	cout << "The sum of your 3 numbers is: " << CalculateSum(155, 940, 168);
+using namespace std;
 
-	/*
-	double num1, num2, num3, sum;
-	cout<< " Enter 3 number to sum:" ;
+int main() {
+	// promptiung user for 3 numbers to add 1/2
+	short num1, num2, num3, sum;
+	cout << "Enter 3 number to sum:" ;
 	cin >> num1 >> num2 >> num3;
+

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add